components/Shimmer/src/Shimmer/Shimmer.xaml

```diff
@@ -0,0 +1,38 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:labs="using:CommunityToolkit.Labs.WinUI">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Dark">
+            <StaticResource x:Key="ShimmerBackground"
+                            ResourceKey="ControlAltFillColorTertiaryBrush" />
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="Light">
+            <StaticResource x:Key="ShimmerBackground"
+                            ResourceKey="ControlAltFillColorTertiaryBrush" />
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="HighContrast">
+            <StaticResource x:Key="ShimmerBackground"
+                            ResourceKey="SystemColorWindowColor" />
+        </ResourceDictionary>
+    </ResourceDictionary.ThemeDictionaries>
+
+    <Style TargetType="labs:Shimmer">
+        <Setter Property="Background" Value="{ThemeResource ShimmerBackground}" />
+        <Setter Property="CornerRadius" Value="4" />
+        <Setter Property="MinWidth" Value="40" />
+        <Setter Property="MinHeight" Value="8" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="labs:Shimmer">
+                    <Border x:Name="Shape"
+                            Background="{TemplateBinding Background}"
+                            CornerRadius="{TemplateBinding CornerRadius}" />
+                </ControlTemplate>
```

Comment:
So little XAML. As another follow-up for the tracking issue, maybe should investigate if we can go the `ContentPresenter` route like we did with Constrained Box and do some funky stuff like that... Trying to think what the most lightweight element we can inherit from would be since `Border` is sealed... As we could take a couple of approaches, but would be nice to remove the XAML template as then would be super light-weight.

ContentPresenter would lend itself to just wrapping something else, though I know our attached shadow like effect route could be interesting as well. Probably different use cases depending on what and how things are being loaded. We'll definitely have to play around more with these ideas in the future.